### PR TITLE
Web version of the Advanced User Guide

### PR DESCRIPTION
**Pull request by teolemon**
Web version of the Advanced User Guide by @bcatelin
You can see a preview at https://world.openfoodfacts.net/advanced-user-guide

### CODE REVIEW

**Comment by VaiTon** [comment]
Why not on the wiki?

**Comment by teolemon** [comment]
It was a Word document that I converted to HTML and cleaned. We can convert it to Wiki syntax, and upload manually each image.
Pro: editable
Con: Will it have as much visibility as on the main website ? More conversion and manual upload work

File: lang/en/texts/advanced-user-guide.html [hunk 1]
```diff
@@ -0,0 +1,1015 @@
...
+  .image {
+    margin-left: 0px;
+    margin-top: 0px;
+    transform: rotate(0rad) translateZ(0px);
+    -webkit-transform: rotate(0rad) translateZ(0px);
+  }
+</style>
+
+<img
+  alt="https://static.openfoodfacts.org/images/misc/openfoodfacts-logo-en-356x300.png"
```
Comment (hangy) [inline lang/en/texts/advanced-user-guide.html:14]: This will be available on OBF and OPFF, too. Do we want to hard-code the domain?

Comment (teolemon) [reply lang/en/texts/advanced-user-guide.html:14]: Just deploying it manually on OFF. I think we need specific guides for OBF and OPFF.

**Pull request merged**